addons/website/static/src/snippets/s_dynamic_snippet/000.js

```diff
@@ -13,6 +13,9 @@ const DynamicSnippet = publicWidget.Widget.extend({
     read_events: {
         'click [data-url]': '_onCallToAction',
     },
+    custom_events: {
+        's_dynamic_snippet_rerender': '_onRerender',
+    },
```

Comment:
Is that needed? Can't you just restart the public widget entirely?

Reply:
I made changed
```js
this.trigger_up('s_dynamic_snippet_rerender')
```
to
```js
this.trigger_up('widgets_start_request', {target: this.$el.closest('.s_dynamic')})
```
does it seem alright to you?

Reply:
If it works, seems better indeed :+1: